third_party/move/move-compiler-v2/tests/checking/typing/recursive_structs.move

```diff
@@ -0,0 +1,25 @@
+address 0x42 {
```

Comment:
nit: having test file names like `recursive_structs` and `recursive_struct` (where both contains multiple recursive structs) can make them difficult to maintain or refer to. More descriptive names would be ideal, but even numbering them `recursive_structs_[1,2]` would be better than existing.

Reply:
done